antiphishing/antiphishing.py

```
@@ -1,35 +1,52 @@
 import contextlib
 import datetime
 import re
-from typing import List
+from typing import List, Optional
 from urllib.parse import urlparse
```

Ruff (0.8.2)

`urllib.parse.urlparse` imported but unused

Remove unused import: `urllib.parse.urlparse`

(F401)